deepform/data/add_features.py

```diff
@@ -98,12 +100,12 @@ def pq_index_and_dir(pq_index, pq_path=None):
     return pq_index, pq_path
 
 
-def process_document_tokens(token_file, dest_file, labels):
+def process_document_tokens(token_file, dest_file, labels, max_token_count):
     """Filter out short tokens, add computed features, and return index info."""
     slug = token_file.stem
-    doc = pd.read_parquet(token_file)
+    doc = pd.read_parquet(token_file).reset_index(drop=True)
```

Comment:
@moredatarequired can you explain why we need this reset_index call here?

Reply:
The tokenized docs have an index that is non-contiguous and repeats indices. Without this line, the slicing below breaks. I've fixed this upstream with adding the `reset_index` call to tokenize_pdfs, but the token files stored in S3 all still have this issue.